app/src/main/java/com/example/roome/RoommateSearcherSetProfileActivity.java

```diff
@@ -75,16 +97,19 @@ protected void onCreate(Bundle savedInstanceState) {
         storage = FirebaseStorage.getInstance();
         storageReference = storage.getReference();
         roommateSearcherUser = new RoommateSearcherUser();
+        newApartment = new Apartment(false, null, null, 2, 0);
+
 
         initializeDateFieldVariablesToFalse();
-        addProfilePic = findViewById(R.id.ib_add_photo);
-        profilePic = findViewById(R.id.iv_missingPhoto);
-        addProfilePic.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                uploadPhotoOnClickAS();
-            }
-        });
+
```

Comment:
אפשר למחוק את זה או שצריך לאחכ?

Reply:
we need it in a different place in this class. I'll remove it for now :)